escape-room/doc/glossary.md

```diff
@@ -0,0 +1,70 @@
+---
+title: "Virtuelle Escape-Rooms in der Lehre - Glossar"
+---
+
+# Akteure
+
+## Designer
+
+Designer erstellen mit dem Dungeon Escape-Rooms zur Erreichung von Zielen aus der Lehre.
+
+## Spieler
+
+## (Spieler-) Charakter
+
+Charaktere sind von Spielern gesteuerte Figuren.
+
+## NPCs
+
+NPCs sind Figuren, die von der Simulation gesteuert werden.
+
+# Geschichte
+
+Die Geschichte ist das, was die Spieler bei dem Durchlauf eines Escape-Rooms erleben.
+
+## Geschehnis
+
+Geschehnisse sind Zustandsänderungen der Geschichte, deren Zeitpunkt und Ablauf vom Designer vorgegeben ist.
+
+> Bsp.: Ein Dschinn erscheint 30 Sekunden nach Spielstart und verlangt von den Charakteren die Preisgabe ihrer Namen.
+
+## Ereignis
+
+Ereignisse werden von Spielern initiiert bringen die Handlung der Geschichte/die Beendigung des Escape-Rooms voran.
+
+TODO "geschehnis" und "ereignis" in anlehnung an die erzähltheorie von lahn und meister. das ereignis ist wichtigere. geschehnisse sind nicht wesentlich.
+
+### (Lern-) Rätsel
+
+Rätsel sind Ereignisse, die auf die Erreichung von Lernzielen zusteuern. Lernziele sind z.B. Inhalte von Lehrveranstaltungen oder Soft-Skills, wie logisches Denken, Problemlösungsstrategien, Kommunikation in der Gruppe und Zusammenarbeit.
+
+TODO wollen wir Rätsel für Soft-Skills namentlich trennen?
+TODO rian hat auch den begriff "aufgabe" eingeworfen, die die charaktere erhalten. sollte der umstand der aufforderung von spieler/charakter hier hervorgehoben werden?
```

Comment:
und carsten hat noch den begriff "puzzle" eingeworfen. 

oder ist puzzle die klammer und ein puzzle setzt sich aus kleineren (atomaren) aufgaben zusammen? oder umgedreht? 

(der vollständigkeit halber.)

Reply:
dein "puzzle" war der ganze escape-room, in dem die rätsel und mechaniken "puzzlesteine" sind, richtig?

Reply:
hmmm, so könnte man das betrachten. man könnte aber auch "puzzle" zu den teilaufgaben sagen, d.h. dann sind die puzzleteile die einzelnen interaktionen in einer mechanik.

Reply:
hilfreiches sinnbild oder maßgebliche kategorie?
das "puzzle" kommuniziert schonmal, dass es teilen besteht. das ist gut. je nachdem, wo wir bei der verschachtelung landen, ist die "flachheit" auch gewünscht. ich würde vielleicht einwerfen, dass die teile vermutlich unerwartet ungleichförmig sein werden.